### DISCUSSION TITLE
v1.0.39.1 MSI Installer Pre-Release

### DISCUSSION THREAD

**Discussion by sylikc** [sylikc/jpegview#44] [zzArchive]
This is a new WiX Installer that I've been working on for the past month.

See discussion here.  https://github.com/sylikc/jpegview/discussions/32#discussioncomment-2083241

It's very rudimentary, but the goal is to get it working just enough to get into winget repository.

<hr /><em>This discussion was created from the release <a href='https://github.com/sylikc/jpegview/releases/tag/'>v1.0.39.1 MSI Installer Pre-Release</a>.</em>

**Comment by sylikc** [comment]
This release has been submitted to the winget repository.  It's been approved and merged.

Look forward to it showing up there! @bitsper2nd

https://github.com/microsoft/winget-pkgs/pull/52204

**Reply by bitsper2nd** [reply]
Thanks! I got it installed through winget in my other machine.
